imod/tests/test_mf6/test_mf6_npf.py

```diff
@@ -293,3 +294,39 @@ def test_npf_from_imod5_vertical_anisotropy(imod5_dataset, tmp_path):
     assert "angle1" not in rendered_npf
     assert "angle2" not in rendered_npf
     assert "angle3" not in rendered_npf
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_npf_from_imod5_settings(imod5_dataset, tmp_path):
+    data = deepcopy(imod5_dataset[0])
+    # throw out kva (=vertical anisotropy array) and ani (=horizontal anisotropy array)
+    data.pop("kva")
+    data.pop("ani")
```

Comment:
Why are these popped? I think users want to see this data be applied to the NPF package in the end.

Reply:
removed the pop's